src/Arcus.Templates.ServiceBus.Topic/Program.cs

```diff
@@ -87,6 +87,8 @@ public static IHostBuilder CreateHostBuilder(string[] args)
                 .MinimumLevel.Debug()
                 .MinimumLevel.Override("Microsoft", LogEventLevel.Information)
                 .Enrich.FromLogContext()
+                .Enrich.WithVersion()
```

Comment:
We should add correlation as well or is this pending a release?

Reply:
See above, same awnser as with the Queue worker.